### PR TITLE
log files are not deleted by deleteRealm

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

**Comment by beeender** [comment]
@realm/java

File: realm/realm-library/src/main/java/io/realm/BaseRealm.java [hunk 1]
```diff
@@ -555,7 +555,7 @@ public RealmSchema getSchema() {
     /**
      * Deletes the Realm file defined by the given configuration.
      */
-    protected static boolean deleteRealm(final RealmConfiguration configuration) {
+    static boolean deleteRealm(final RealmConfiguration configuration) {
```
Comment (cmelchior) [inline realm/realm-library/src/main/java/io/realm/BaseRealm.java]: I would prefer this as protected. It should only be called from sub-classes.

**Comment by cmelchior** [comment]
Apart from small comment :+1:

File: realm/realm-library/src/main/java/io/realm/BaseRealm.java (hunk 1 above)
Comment (beeender) [inline realm/realm-library/src/main/java/io/realm/BaseRealm.java]: `no modifier` is more strict i think? Even user extend the class, it still cannot be called.

Comment (cmelchior) [inline realm/realm-library/src/main/java/io/realm/BaseRealm.java]: Ah yes, sorry. Ignore me :) https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html

**Comment by zaki50** [comment]
:+1:

**Pull request merged**